gammapy/cube/fit.py

```diff
@@ -201,9 +202,8 @@ def apply_exposure(self, flux):
 
         For now just divide flux cube by exposure
         """
-        npred = Map.from_geom(self.geom, unit="")
-        npred.data = (flux * self.exposure.quantity).to("").value
-        return npred
+        npred = (flux * self.exposure.quantity).to_value("")
```

Comment:
With map arithmetics there is an additional copy of the data involved, which I'd like to avoid here, because it is called many times during a fit. I'll keep as is for now.